File: src/main/java/com/cfblj/carrental/service/CarRentalService.java
package com.cfblj.carrental.service;

import com.baomidou.mybatisplus.extension.service.IService;
import com.cfblj.carrental.model.CarRental;
import com.cfblj.carrental.utils.Pages;

public interface CarRentalService extends IService<CarRental> {
    Pages getCarRentalPage(int curPage, int size);

    CarRental getCarRentalById(String id);
}

File: src/main/resources/static/js/carrental/service/carRentalService.js
app.service('carRentalService', function ($http) {


    this.getCarRentalPage=function(curPage, size){
        return $http.get('/carRental/getCarRentalPage?curPage='+curPage+'&size='+size);
    }

    this.getCarRentalById=function (id) {
        return $http.get('/carRental/getCarRentalById?id='+id);
    }

    // this.addStock=function (stock) {
    //     return $http.post('/carRental/addStock', stock);
    // }
    //
    // this.updateStock=function (stock) {
    //     return $http.post('/carRental/updateStock', stock);
    // }
})